components/IscnRegisterForm.vue

```diff
@@ -1114,38 +1115,52 @@ export default class IscnRegisterForm extends Vue {
   }
 
   async mounted() {
-  if (this.epubMetadata) {
-    this.name = this.epubMetadata.title;
-    this.description = this.extractText(this.epubMetadata.description);
-    this.author.name = this.epubMetadata.author;
-    this.author.authorDescription = 'Author'
-    this.language = this.epubMetadata.language
-    this.tags = this.epubMetadata.tags
-    this.thumbnailUrl = this.formatArweave(this.epubMetadata.thumbnailUrl) as string
-    if (this.author.name) {
-      this.authors.push(this.author)
+    this.uploadStatus = 'loading'
+
+    if (this.epubMetadata) {
+      this.name = this.epubMetadata.title;
+      this.description = this.extractText(this.epubMetadata.description);
+      this.author.name = this.epubMetadata.author;
+      this.author.authorDescription = 'Author'
+      this.language = this.epubMetadata.language
+      this.tags = this.epubMetadata.tags
+      this.thumbnailUrl = this.formatArweave(this.epubMetadata.thumbnailUrl) as string
+      if (this.author.name) {
+        this.authors.push(this.author)
+      }
+    }
+
+    if (this.address) {
+      const iscnOwner = await this.fetchUserInfoByAddress(this.address)
+      this.authors.push(iscnOwner)
     }
+    // ISCN Fee needs Arweave fee to calculate
+    await this.calculateISCNFee()
+    this.uploadStatus = ''
   }
-  if (this.address) {
-    const iscnOwner = {
-      name: this.$t('iscn.meta.stakeholders.name.placeholder') as string,
-      wallet: [{
-        content: this.address,
-        id: 1,
-        type: 'like',
-        isOpenOptions: false,
-      }],
-      url: [],
-      likerId: '',
-      authorDescription: 'ISCN owner',
+
+  async fetchUserInfoByAddress(address: any) {
+    try {
+      const { data } = await this.$axios.get(getUserInfoMinByAddress(address))
+      return {
+        name: data?.displayName || address,
+        wallet: [{ content: address, id: 1, type: 'like', isOpenOptions: false }],
+        url: [],
+        likerId: data?.user || '',
+        authorDescription: data?.description || 'Publisher',
+      }
+    } catch (error) {
+      // eslint-disable-next-line no-console
+      console.error(error)
+      return {
+        name: address,
+        wallet: [{ content: address, id: 1, type: 'like', isOpenOptions: false }],
+        url: [],
+        likerId: '',
+        authorDescription: 'Publisher',
+      }
     }
```

Comment:
```suggestion
    let userData: any = null;
    try {
      ({ data: userData } = await this.$axios.get(getUserInfoMinByAddress(address)))
    } catch (error) {
      // eslint-disable-next-line no-console
      console.error(error)
    }
    return {
      name: userData?.displayName || address,
      wallet: [{ content: address, id: 1, type: 'like', isOpenOptions: false }],
      url: [],
      likerId: userData?.user || '',
      authorDescription: userData?.description || 'Publisher',
    }
```

Reply:
[🎨 Handle default info when user data is unavailable](https://github.com/likecoin/app-like-co/pull/440/commits/4474a883bc8b1d79c1d50380cd17c25a05fdfcfa)